src/components/ClipboardTextfield.tsx

```diff
@@ -32,12 +36,18 @@ export default function ClipboardTextfield({
         onClick={onClickCopy}
       >
         <div className={`select-none`}>
+          {/* TODO develop custom tooltip component */}
+          {text}&nbsp;&nbsp;
```

Comment:
The copy ip button should have the text to the right, and icon to the left.
Maybe make `iconSide` a prop too?

Also the icon currently have `mr-2` but if the text is on the left, the space is in the wrong place. please look into tailwindcss `gap` or `space`

![image](https://user-images.githubusercontent.com/44907675/221375327-4d385837-0347-4546-841c-52ba4d1e0b79.png)